### ISSUE TITLE
What's your work's logic to make CLP benchmark based on TP.

### ISSUE THREAD

**Issue by WoozieDevCUG** [bdi-lab/BiVE#1]
I think the CLP datas are based on TP,but the ratio of numbers of Train/Test/Valid on CLP is not 8:1:1.I noticed that the number of CLP Test datas is 3494 which is equal to TP,but the Train and Valid numbers of CLP are far from TP.

**Comment by ChanyoungChung98** [comment]
Sorry for the late reply.

`train.txt` and `valid.txt` in both `conditional` and `conditional_head` folders are only used in the filtering process during evaluation.

To make use of the `OpenKE` implementation, we store each higher-level triplet $\langle(h_1, r_1, t_1), R, (h_2, r_2, t_2)\rangle$ in the form of $(h_2, (h_1, t_1, r_1, R, r_2), t_2)$. For efficiency, we removed all higher-level triplets in training and validation sets that do not share $(h_1, t_1, r_1, R, r_2)$ with any of the higher-level triplets in the test set.

Similarly, we store each higher-level triplet $\langle(h_1, r_1, t_1), R, (h_2, r_2, t_2)\rangle$ in the form of $(h_1, (r_1, R, h_2, t_2, r_2), t_1)$, and we removed all higher-level triplets in training and validation sets that do not share $(r_1, R, h_2, t_2, r_2)$ with any of the higher-level triplets in the test set.

Sincerely,
Chanyoung Chung
